packages/mason_cli/lib/src/commands/bundle.dart

```diff
@@ -63,81 +63,97 @@ class BundleCommand extends MasonCommand {
   @override
   Future<int> run() async {
     final source = results['source'] as String;
+    final outputDir = results['output-dir'] as String;
+    final bundleType = (results['type'] as String).toBundleType();
+
+    final bricks = _parseBricks(source);
+    final tempBricksJson = <BricksJson>[];
+    final bundleProgress = logger.progress('Bundling ${bricks.length} bricks');
+
+    final bundlePaths = <String>[];
+    try {
+      for (final brick in bricks) {
+        final Directory brickDirectory;
+        if (brick.location.path != null) {
+          brickDirectory = Directory(brick.location.path!);
+        } else {
+          final tempBrickJson = BricksJson.temp();
+          final cachedBrick = await tempBrickJson.add(brick);
+          tempBricksJson.add(tempBrickJson);
+          brickDirectory = Directory(cachedBrick.path);
+        }
+
+        if (!brickDirectory.existsSync()) {
+          throw BrickNotFoundException(brickDirectory.path);
+        }
+
+        final bundle = createBundle(brickDirectory);
+        bundleProgress.update('Bundling ${bundle.name}');
+
+        final String bundlePath;
+        switch (bundleType) {
+          case BundleType.dart:
+            bundlePath = await _generateDartBundle(bundle, outputDir);
+            break;
+          case BundleType.universal:
+            bundlePath = await _generateUniversalBundle(bundle, outputDir);
+            break;
+        }
+        bundleProgress.update('Bundled ${bundle.name}');
+        bundlePaths.add(bundlePath);
+      }
 
-    final Brick brick;
+      logger.info('${lightGreen.wrap('✓')} Generated ${bricks.length} files:');
```

Comment:
Because the `bundleProgress` is still running this log might not be placed correctly, instead we should do a `bundleProgress.complete('message here')` instead the info.

Reply:
or call bundleProgress.complete before this line?